packages/websocket/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@atproto/websocket",
+  "version": "0.0.1",
```

Comment:
I might be wrong but I think that even new packages need a changeset to be released by our ci.